### PR TITLE
Add authorized endpoint to verify if user is granted to access the IDP client.

### PR DESCRIPTION
**Pull request by alejandrodevs**
## Addresses issue: [#TOOL-31](https://amcoit.atlassian.net/browse/TOOL-31)

Add authorized endpoint to verify if user is granted to access the IDP client.

### CODE REVIEW

**Review by MtzRojas — APPROVED** [review]
:shipit:

File: lib/generators/gamora/templates/gamora.rb [hunk 1]
```diff
@@ -22,4 +22,5 @@
   # config.allow_create = true
   # config.userinfo_cache_expires_in = 1.minute
   # config.introspect_cache_expires_in = 0.seconds
+  # config.authorization_method = -> (user) { user.present? }
```
Comment (javierg) [inline lib/generators/gamora/templates/gamora.rb]: IMO to better show the intention of the feature, I will sample here with something like:

```
# config.authorization_method = -> (user) { user.teacher? }
```

or something that indicates better that we are looking for authorization

Comment (alejandrodevs) [reply lib/generators/gamora/templates/gamora.rb]: Done, please take another look.

File: README.md [hunk 2]
```diff
@@ -132,6 +132,23 @@ Gamora.setup do |config|
...
+
+In order to inform if a user's access token is granted to access the IDP
+client, it is possible to configure the authorization method in the initializer
+that will be used in the `/auth/amco/authorized` endpoint.
+
+```ruby
+Gamora.setup do |config|
+  ...
+
+  config.authorization_method = -> (user) { MyAuthorizationService.call(user) }
```
Comment (javierg) [inline README.md:145]: 👍

**Review by javierg — APPROVED** [review]
:shipit:

**Pull request merged**